### ISSUE TITLE
`fill_opacity=0` not working with `MarkupText` in Manim v0.4.0

### ISSUE THREAD

**Issue by dipamsen** [ManimCommunity/manim#1084]
## Description of bug / unexpected behavior
`MarkupText` with `fill_opacity=0` renders with fill.

## Expected behavior
This used to work correctly in Manim v0.3.0, but doesn't work in v0.4.0
There should be no fill in `fill_opacity=0` `MarkupText`

## How to reproduce the issue

<details open><summary>Code for reproducing the problem</summary>

```py
class Bug(Scene):
  def construct(self):
    t = MarkupText("<b>Stroked Text</b>", fill_opacity=0, stroke_width=2)
    self.play(Write(t))
```

</details>

## Additional media files

<details open><summary>Images/GIFs</summary>

### Version 0.3 Output

https://user-images.githubusercontent.com/59444569/110228086-5239c500-7f24-11eb-8b9c-447c8088e8e6.mp4

### Version 0.4 Output

https://user-images.githubusercontent.com/59444569/110228180-19e6b680-7f25-11eb-95c1-4f1e44faf075.mp4

</details>

**Comment by MrMallIronmaker** [comment]
Thanks for filing this issue! I saw this convo on Discord but it was old so I came here.

This is definitely SVG parser stuff. For context, it's been discussed in a couple of places (here in https://github.com/ManimCommunity/manim/pull/1058#issuecomment-787478624 and touched on in #1023, but not explained as well). The underlying issue is that SVGMobject has no way of knowing whether the values were passed in by the user (as in your case) or passed in as VMobject defaults (which should be overridden by the SVG defaults).  

A workaround for now is to set the values after creation, i.e;
```py
class Bug(Scene):
    def construct(self):
        t = MarkupText("<b>Stroked Text</b>")
        t.set_fill(opacity=0)
        t.set_stroke(width=2)
        self.play(Write(t))
```

https://user-images.githubusercontent.com/1994318/110343586-80310f00-7fe1-11eb-9cfb-74946bb6d5be.mp4

This is definitely a bug though, and hopefully will be addressed in the next release.
